specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/examples/StorageAccountAbortHnsOnMigration.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res4228",
+    "accountName": "sto2434",
+    "api-version": "2021-06-01"
+  },
+  "responses": {
+    "200": {},
+    "202": {}
+  }
+}
```

Comment:
Abort HNSon Migration always fail as 400 in my test, see network trace below.
Step:
1. invoke migration with type "HnsOnHydrationRequest"
2. wait for 10 s
3. invoke about migration request , fail with 400

```
POST https://management.azure.com/subscriptions/{SubId}/resourcegroups/res4933/providers/Microsoft.Storage/storageAccounts/sto7451/hnsonmigration?api-version=2021-06-01&requestType=HnsOnHydrationRequest HTTP/1.1
x-ms-client-request-id: 22d0615f-92b2-4450-9ec3-ee9b4afa2b11
Accept-Language: en-US
Authorization: Bearer ***
User-Agent: FxVersion/4.6.30321.06 OSName/Windows OSVersion/Microsoft.Windows.10.0.19043. Microsoft.Azure.Management.Storage.StorageManagementClient/22.0.0.0
Content-Length: 0
Host: management.azure.com


HTTP/1.1 202 Accepted
Cache-Control: no-cache
Pragma: no-cache
Content-Type: text/plain; charset=utf-8
Expires: -1
Location: https://management.azure.com/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/providers/Microsoft.Storage/locations/centraluseuap/asyncoperations/7001fe91-53d0-4aaf-a045-edfa9cb81583?monitor=true&api-version=2021-06-01
Retry-After: 17
Azure-AsyncOperation: https://management.azure.com/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/providers/Microsoft.Storage/locations/centraluseuap/asyncoperations/7001fe91-53d0-4aaf-a045-edfa9cb81583?monitor=true&api-version=2021-06-01
x-ms-request-id: 7001fe91-53d0-4aaf-a045-edfa9cb81583
Strict-Transport-Security: max-age=31536000; includeSubDomains
Server: Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0 Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-writes: 1195
x-ms-correlation-request-id: 60df778c-8b05-4096-8515-dad1150d84e6
x-ms-routing-request-id: SOUTHEASTASIA:20210816T100136Z:60df778c-8b05-4096-8515-dad1150d84e6
X-Content-Type-Options: nosniff
Date: Mon, 16 Aug 2021 10:01:36 GMT
Content-Length: 0



------------------------------------------------------------------
POST https://management.azure.com/subscriptions/{SubId}/resourcegroups/res4933/providers/Microsoft.Storage/storageAccounts/sto7451/aborthnsonmigration?api-version=2021-06-01 HTTP/1.1
x-ms-client-request-id: 7b59fb82-ea06-4787-839d-f5083e9f9d59
Accept-Language: en-US
Authorization: Bearer ***
User-Agent: FxVersion/4.6.30321.06 OSName/Windows OSVersion/Microsoft.Windows.10.0.19043. Microsoft.Azure.Management.Storage.StorageManagementClient/22.0.0.0
Content-Length: 0
Host: management.azure.com


HTTP/1.1 400 Bad Request
Cache-Control: no-cache
Pragma: no-cache
Content-Length: 106
Content-Type: application/json
Expires: -1
x-ms-request-id: 4fcd9f35-0f1d-4ec9-988e-35b290485c22
Strict-Transport-Security: max-age=31536000; includeSubDomains
Server: Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0 Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-writes: 1194
x-ms-correlation-request-id: a46cade3-e62f-469f-b877-b34b3f8c3aa0
x-ms-routing-request-id: SOUTHEASTASIA:20210816T100146Z:a46cade3-e62f-469f-b877-b34b3f8c3aa0
X-Content-Type-Options: nosniff
Date: Mon, 16 Aug 2021 10:01:46 GMT

{"error":{"code":"HnsMigrationNotFound","message":"Hns migration for the account: sto7451 is not found."}}

------------------------------------------------------------------
```